vendor/rachidlaasri/laravel-installer/src/Helpers/DatabaseManager.php

```diff
@@ -44,8 +44,7 @@ private function migrate($outputLog)
             Artisan::call('migrate', ["--force"=> true], $outputLog);
         }
         catch(Exception $e){
-             throw new Exception('Cannot Establish Database Conection.Check Details');
-            // return $this->response($e->getMessage());
+             throw new Exception($e->getMessage());
```

Comment:
its inside vendor. Are you sure about it?

Reply:
Yes, later we can improve the installer and handle all functions from Billing's core files